client/app/components/game-client.js

```diff
@@ -0,0 +1,64 @@
+import Ember from 'ember';
+import GameChannel from 'vilive/utils/game-channel';
+
+export default Ember.Component.extend({
+  init() {
+    this.set('messages', Ember.makeArray());
+    GameChannel.init(this.get('session-manager').currentToken());
+    GameChannel.on('event', payload => {
+      this.putMessage(true, payload.body);
```

Comment:
I think we can go further and have the server send a standard message response that we can consume. The boolean was not my first attempt I basically had what you just suggested but for templating the messages it was needed. The if/unless statements in the template only work on boolean values so I could not do `{{if origin="player" ...}}`.

I saw you note about the styling of the messages as well, this will fade away I think if we restyle and have the server deliver a standard message that maybe is in that form.